spec/controllers/medications_controller_spec.rb

```diff
@@ -35,5 +35,23 @@
         expect(subject).to eq('<div class="small_margin_top"><i class="fa fa-bell small_margin_right"></i>Refill reminder email, Daily reminder email</div>')
       end
     end
+
+    describe 'DELETE #destroy' do
+      let(:user) { FactoryGirl.create(:user1) }
+      let!(:medication) { FactoryGirl.create(:medication, userid: user.id) }
+
+      context 'when the user is logged in' do
```

Comment:
This is sensational but would you mind also testing what happens when the user isn't logged in?

Reply:
Alrighty! I added a test similar to the this [category#destroy test](https://github.com/julianguyen/ifme/blob/master/spec/controllers/categories_controller_spec.rb#L185).